interface/patient_file/encounter/diagnosis.php

```diff
@@ -1,312 +0,0 @@
-<?php
```

Comment:
LibreEHR\interface\super\rules\library\RuleCriteriaDiagnosis.php may be calling this file.